Vagrantfile

```diff
@@ -0,0 +1,40 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/bionic64"
+
+  # Setup machine
+  config.vm.provision "shell", privileged: true, inline: <<-SHELL
+    apt install build-essential -y
+  SHELL
+
+  # Install miniconda
```

Comment:
Do we need to setup conda for the VM?

Reply:
Maybe we don't need to, but I tried to mimic the env setup that is suggested in the documentation. How would you install all the tools and dependencies without conda?

Reply:
Why not just use the system python?

Reply:
I think we could but I thought conda was also used to have multiple envs with different python versions

eg
```bash
conda create -n py3.6 python=3.6
conda create -n py3.7 python=3.7
```

How do you currently run tests locally for different python versions?

Reply:
Can you check this would work if you were to drop the conda install and just use the python version installed on the vm image? 

What is the python version in the ubuntu/bionic image?